sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/aio/client.py

```diff
@@ -141,24 +120,27 @@ class SecretClient(AsyncKeyVaultClientBase):
                 :caption: Updates a secret's attributes
                 :dedent: 8
         """
-        enabled = kwargs.pop('enabled', None)
-        if enabled is not None or not_before is not None or expires is not None:
-            attributes = self._client.models.SecretAttributes(enabled=enabled, not_before=not_before, expires=expires)
+        enabled = kwargs.pop("enabled", None)
+        not_before = kwargs.pop("not_before", None)
+        expires_on = kwargs.pop("expires_on", None)
+        if enabled is not None or not_before is not None or expires_on is not None:
+            attributes = self._client.models.SecretAttributes(
+                enabled=enabled, not_before=not_before, expires=expires_on
+            )
         else:
             attributes = None
         bundle = await self._client.update_secret(
             self.vault_endpoint,
             name,
             secret_version=version or "",
-            content_type=content_type,
             secret_attributes=attributes,
             error_map=_error_map,
             **kwargs
         )
         return SecretProperties._from_secret_bundle(bundle)  # pylint: disable=protected-access
 
     @distributed_trace
-    def list_secrets(self, **kwargs: "**Any") -> AsyncIterable[SecretProperties]:
+    def list_properties_of_secrets(self, **kwargs: "Any") -> AsyncIterable[SecretProperties]:
```

Comment:
I think AsyncIterable should be in quotes here and in other instances in this file

Reply:
I agree, but am leaving it for a mypy cleanup pass because it will involve larger changes to the file.